src/components/EntityOpsLevelMaturityContent.tsx

```diff
@@ -59,10 +60,92 @@ export const EntityOpsLevelMaturityContent = () => {
   }
 
   const { maturityReport } = service;
-  const levels = opsLevelData.account?.rubric?.levels?.nodes;
-  const levelCategories = opsLevelData.account?.service?.maturityReport?.categoryBreakdown;
-  const checkResultsByLevel = opsLevelData.account?.service?.serviceStats?.rubric?.checkResults?.byLevel?.nodes;
-  const checkStats = opsLevelData.account?.service?.checkStats;
+  const levels = opsLevelData.account.rubric?.levels?.nodes;
+  const levelCategories = opsLevelData.account.service.maturityReport?.categoryBreakdown;
+  const checkResultsByLevel = opsLevelData.account.service.serviceStats?.rubric?.checkResults?.byLevel?.nodes;
+  const scorecards = opsLevelData.account.service.serviceStats?.scorecards?.nodes;
+  // const checkStats = opsLevelData.account.service.checkStats;
+  const scorecardCategories = scorecards ? scorecards.reduce(
+    (scoreCardAccumulator: LevelCategory[], currentScorecard) => {
+      if (!currentScorecard.categories?.edges) return scoreCardAccumulator;
+      return [
+        ...scoreCardAccumulator,
+        ...currentScorecard.categories.edges.reduce(
+          (nodeAccumulator: LevelCategory[], currentEdge): LevelCategory[] => [
+            ...nodeAccumulator,
+            {
+              category: {name: currentEdge.node?.name ?? ''},
+              level: currentEdge.level?.name ? {name: currentEdge.level?.name} : null,
+              rollsUp:
+                  currentScorecard?.scorecard?.affectsOverallServiceLevels,
+            },
+          ],
+          [],
+        ),
+      ];
+    },
+    [],
+  ).sort((a, b) => (a.category.name < b.category.name ? -1 : 1)) : undefined;
+  // const scorecardCategories: LevelCategory[] = []
+
+
+  function checksByLevelIncludingScorecards() {
+    const result = cloneDeep(checkResultsByLevel);
+
+    result?.forEach((checkResults) => {
+      // Use level's 'index' field b/c we don't have level ID here. Note: 'index' *is not* the same as array index
+      const levelIndex = checkResults.level.index;
+
+      scorecards?.forEach((scorecard) => {
+        const entry = scorecard.checkResults?.byLevel?.nodes?.find(
+          (node) => node.level.index === levelIndex,
+        );
+        if (!entry) return;
+
+        entry.items.nodes.forEach((node) => {
+          node.check.isScorecardCheck = true;
+        });
+        checkResults.items.nodes = [
+          ...checkResults.items.nodes,
+          ...entry.items.nodes,
+        ];
+      })
+    })
+    return result || [];
+  }
```

Comment:
All of this stuff is untested (but stolen from OpsLevel) and we should work on componentizing it next week.